packages/typespec-azure-resource-manager/lib/common-types/types.tsp

```diff
@@ -298,7 +298,7 @@ model ErrorAdditionalInfo {
 
   /** The additional info. */
   @visibility(Lifecycle.Read)
-  info?: {};
+  info?: Record<unknown>;
```

Comment:
It's breaking change to SDK. Why not just keep our previous decision?

Reply:
I think its a breaking change in the SDK eitherway. JS generates that currently as `Record<string, unknown>` because they did respect the `type: object`

Regardless of that I do agree that I think this is supposed to be `unknown` anyway because there is a `type` right next to it that woudl seem to indicate the type of this property. However this is not super clear hence the comment from mark looking into more investigation